config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,15 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+#: --tmpfs parameter
+tmpfs_path = /run
+run_options_csv = --tmpfs,%(tmpfs_path)s
+#: command to run in container to check if passed --tmpfs is mounted as tmpfs type
+fndmnt = findmnt -n -t tmpfs %(tmpfs_path)s
+cmd = '%(fndmnt)s'
```

Comment:
Don't let the quotes bite you either, they are preserved right down through execution.  I'm not sure off-hand if they're needed here or not, but I've certainly been tripped up myself by adding them into ``ini`` file when there were not needed.